src/main/java/com/artipie/Pie.java

```diff
@@ -77,7 +77,7 @@ public Pie(final Settings settings, final Vertx vertx) {
     @SuppressWarnings("PMD.OnlyOneReturn")
     public Response response(final String line, final Iterable<Map.Entry<String, String>> headers,
         final Publisher<ByteBuffer> body) {
-        Logger.info(this, "Request: %s", line);
+        Logger.info(this, "Request: %s", line.replace("\r\n", ""));
```

Comment:
@Sammers21 you can just use `line.strip()` instead to remove all white-space chars from the end